avocado/utils/multipath.py

```diff
@@ -97,6 +97,24 @@ def get_mpath_name(wwid):
         return process.run(cmd, sudo=True).stdout_text.split()[0]
 
 
+def get_mapper_name(dev_mapper):
+    """
+    Get the mpath name for given device mapper
+
+    :param dev_mapper: Input device mapper dmX
+    :return: Multipath mpath name mpathX
```

Comment:
can be as below
:return:  mpath name like mpathX

Reply:
done